source/features/quick-repo-deletion.tsx

```diff
@@ -64,7 +65,10 @@ async function buttonTimeout(buttonContainer: HTMLDetailsElement): Promise<boole
 		buttonContainer.open = false;
 	}, {once: true});
 
-	void verifyScopesWhileWaiting(abortController);
```

Comment:
This change is unrelated (and should be reverted). You are making the feature wait for scope verification, which is currently asynchronous.

Reply:
The idea behind this change is so the button won't change intro `Deleting repo in` so there is less visual mess going on when token doesn't have the scope

Reply:
The mess is fine as it's only supposed to happen once, if misconfigured. This just makes the operation slower every time instead.